spec/views/application_view_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationView, type: :view do
+  it { is_expected.to be_a(Stache::Mustache::View) }
+
+  describe '#i18n' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.